src/terraform/linode/terraform.tf

```
@@ -0,0 +1,11 @@
+terraform {
+	required_providers {
+		linode = {
+			source	= "linode/linode"
```

IMO: I want to fix linode provider version in the same as the terraform version.

```suggestion
			source	= "linode/linode"
			version = "~> 2.35.0"
```

---

added!